src/NUnitFramework/tests/Constraints/CollectionContainsConstraintTests.cs

```diff
@@ -51,7 +51,7 @@ public void CanTestContentsOfObjectList()
             Assert.That(list, new CollectionContainsConstraint(item));
         }
 
-#if !SILVERLIGHT
+#if !SILVERLIGHT && !PORTABLE
```

Comment:
I'm wondering about all the !PORTABLE you are having to add. Were these files that were simply not included in the portable build before? If not, how come they stopped working?